slack-api-client/src/test/java/test_with_remote_apis/methods/chat_Test.java

```diff
@@ -43,1026 +47,1288 @@
+                                .token(botToken)
+                                .channel(randomChannelId)
+                                .text(null)
+                                .blocksAsString(blocksAsString)
+                                .build());
+                assertThat(postResponse.getError(), is(nullValue()));
+
+                ChatUpdateResponse updateMessage = slack.methods().chatUpdate(ChatUpdateRequest.builder()
+                                .channel(randomChannelId)
+                                .token(botToken)
+                                .ts(postResponse.getTs())
+                                .text(null)
+                                .blocksAsString(blocksAsString)
+                                .build());
+                assertThat(updateMessage.getError(), is(nullValue()));
+        }
+
+        @Test
+        public void issue_785() throws Exception {
+                loadRandomChannelId();
+
+                String ts = null;
+                try {
+                        ChatPostMessageResponse newMessage = slack.methods(botToken).chatPostMessage(r -> r
+                                        .text("Here is the link: https://i.imgur.com/brgYmPX.gifv")
+                                        .unfurlLinks(true)
+                                        .unfurlMedia(true)
+                                        .channel(randomChannelId));
+                        assertThat(newMessage.getError(), is(nullValue()));
+                        ts = newMessage.getTs();
+                        Thread.sleep(3_000L);
+
+                        ConversationsHistoryResponse history = slack.methods(botToken).conversationsHistory(r -> r
+                                        .channel(randomChannelId).limit(1));
+                        assertThat(history.getError(), is(nullValue()));
+                        List<Attachment> attachments = history.getMessages().get(0).getAttachments();
+                        // the attachments can be null
+                        if (attachments != null && attachments.get(0) != null) {
+                                Attachment.VideoHtml videoHtml = attachments.get(0).getVideoHtml();
+                                assertThat(videoHtml.getHtml(), is(
+                                                "<video muted loop controls autoplay preload=\"auto\" playsinline height=\"540\" width=\"960\" poster=\"https://i.imgur.com/brgYmPXh.jpg\" src=\"https://i.imgur.com/brgYmPX.mp4\"></video>"));
+                                assertThat(videoHtml.getSource(), is(nullValue()));
+                        }
+                } finally {
+                        if (ts != null) {
+                                final String _ts = ts;
+                                slack.methods(botToken).chatDelete(r -> r.channel(randomChannelId).ts(_ts));
+                        }
                 }
-            }
         }
-    }
-
-    String blocksAsString = "[\n" +
```

Comment:
Looks like my formatter in VSCode updated this whole file; is there a particular formatter I should be using? (My current default is `Language Support for Java(TM) by Red Hat`)